app/helpers/comments_helper.rb

```diff
@@ -2,7 +2,12 @@ module CommentsHelper
   def comments_tree_for(comments)
     comments.map do |comment, nested_comments|
       render 'comments/comment', comment: comment do
-        content_tag(:div, comments_tree_for(nested_comments), class: 'reply') if nested_comments.size > 0
+        content_tag(
+          :div,
+          comments_tree_for(nested_comments),
+          class: "comments",
+          'data-commentable-id' => comment.id
```

Comment:
just out of understanding best practice, do you choose to create a custom data-id field instead of using the id field just to be specific?